### PR TITLE
change buf value type

### PR DESCRIPTION
**Pull request by tranma**
a better vbuf

### CODE REVIEW

File: test/cli/repl/t10-avalanche/expected [hunk 1]
```diff
@@ -45,10 +45,7 @@ conv$9 =
     \conv$10 : Buf Int.  
     \conv$11 : Int. 
     Latest_push# [Int] conv$10
-    conv$11)
-    ([ExceptScalarVariableNotAvailable
-     ,ExceptScalarVariableNotAvailable
-     ,ExceptScalarVariableNotAvailable] : Buf Int)
+    conv$11) (Buf 3 [] : Buf Int)
```
Comment (amosr) [inline test/cli/repl/t10-avalanche/expected]: yeah, I like this much better

**Comment by amosr** [comment]
:minidisc:

**Comment by jystic** [comment]
:8ball:

**Comment by tranma** [comment]
travis you weird, oh well

**Comment by jystic** [comment]
travis is driving me nuts today

**Pull request merged**